### PR TITLE
Lint all code, not just tests, and fix issues.

### PR DESCRIPTION
**Pull request by avindra**
This bumps `eslint` and `request`.  The other version bumps are just to bump explicitly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,20 +36,20 @@
...
   },
   "dependencies": {
     "extract-zip": "~1.5.0",
-    "fs-extra": "~0.26.4",
+    "fs-extra": "~0.26.7",
     "hasha": "^2.2.0",
     "kew": "~0.7.0",
     "progress": "~1.1.8",
-    "request": "~2.67.0",
+    "request": "~2.71.0",
```
Comment (marti4ka) [inline package.json]: Yes, I definitely need this. Why isn't it merged yet?

**Comment by avindra** [comment]
Has anyone reviewed this yet? @nicks

**Comment by nicks** [comment]
thanks! out of curiosity, what do you need this for? i thought it was non-essential clean-up, and could wait until the next time we need to do a version update.

**Comment by marti4ka** [comment]
Hi @nicks and thanks for merging. We are using Snyk and we had a vulnerability because of the `request` version and the patch didn't work properly. More specifically: 
```
✗ Medium severity vulnerability found on request@2.67.0
- desc: Remote Memory Exposure
- info: https://snyk.io/vuln/npm:request:20160119
- from: bridge@0.0.1 > phantomjs@1.9.20 > request@2.67.0
No direct dependency upgrade can address this issue.
```

**Pull request merged**